src/components/modals/TransferPDNTModal/TransferPDNTModal.tsx

```diff
@@ -34,7 +35,7 @@ function TransferPDNTModal({
   const isMobile = useIsMobile();
   const [accepted, setAccepted] = useState<boolean>(false);
   const [toAddress, setToAddress] = useState<string>('');
-  const [isValidAddress, setIsValidAddress] = useState<boolean>(false);
+  const [isValidAddress, setIsValidAddress] = useState(false);
```

Comment:
still don't think we need this - just use the warning from the Validation Input and the message associated with it

Reply:
this is to disable the `next` button

Reply:
that should be determined by validity - not this